scene/gui/color_picker.h

```diff
@@ -93,14 +120,19 @@ class ColorPicker : public BoxContainer {
 	HSeparator *preset_separator = memnew(HSeparator);
 	Button *btn_add_preset = memnew(Button);
 	Button *btn_pick = memnew(Button);
-	CheckButton *btn_hsv = memnew(CheckButton);
-	CheckButton *btn_raw = memnew(CheckButton);
-	HSlider *scroll[4];
-	SpinBox *values[4];
-	Label *labels[4];
+
+	OptionButton *mode_option_button = memnew(OptionButton);
+
+	HSlider *scrolls[SLIDER_COUNT];
```

Comment:
Why is this called `scrolls` if it holds sliders and everywhere it's called sliders?

Reply:
Don't know, it was like this before.

Reply:
Then it can be renamed.